about.html

```diff
@@ -0,0 +1,131 @@
+        </ul>
+      </div>
+
+
+    </div>
+    <!-- <div class="logo"><img src="./images/Restaurant-logo.jpg" alt=""></div> -->
+  </nav>
+<header>
+  <h1 class="open-comments first-line">HEY YOU THERE COME TASTE SOME!</h1>
+  <h2 class="welcome message">WELCOME TO CHEF YOUNG'S RESTAURANT AKA THE MOST FAMOUS AND SUMPTUOUS RESTAURANT IN THE WORLD OF 2023</h2>
+  <hr class="line">
+  <p class="paragraph">The menu features classic French cuisine, with a focus on fresh, seasonal ingredients. The chef is passionate about his food, and it shows in the dishes he creates. The portions are generous, and the flavors are always well-balanced.</p>
+  </header>
+
+  <section class="section-paragraph">
+    <p class="first-section-paragraph">This year, we are anticipating bringing out new recipies, If you don't wanna miss out email us on</p>
+    <h4 class="email">chefyoung2023@gmail.com</h4>
+  </section>
+
+  <section>
+    <h2 class="chef">Chef Young Restaurant logo of 2023</span> </h2>
+    <hr class="line line-resto">
+    <p class="voting votting-logo">This logo of chef Young Restaurant was voted by from  January 1st to January 31st. After online public voting and selection committe's judging process, The logo from Young Gaius won the prize. After consultation with Mr. Gaius, we optimized the prize winning logo for our restaurant:</span></p>
+    <div class="picture logo-picture">
+        <img class="image" src="images/main-logo.jpg" alt="Restaurant-logo">
+    </div>
+    <hr class="h-rule">
+
+    <div class="recent-dishes">
+    <h2 class="see-recent">See recent diches</h2>
+    <hr class="line ligne">
+    <p class="burgers-pizz">Take a look at the two recent output of our latest recipies. Right here below we have a classic dish of pizza and buggers, so yummy!</p>
+    </div>
+    <div class="double-picture">
+      <div class="de-phote">
+        <div class="de-phote-content">
+          <h4> Dinner night menu</h4>
+          <p>Wednesday @8:00pm</p>
+        </div>
+          
+        
+      </div>
+      <div class="de-phote">
+        <div class="de-phote-content">
+          <h4> Burgers Nights</h4>
+          <p>Friday's @8:00pm</p>
+        </div>
+        
+      </div>
+   </div>
+  </section>
+
+  <section class="last-section l-s">
+    <div class="sponsors">
+        <h3 class="partner">Partner</h3>
+    </div>
+    <hr class="line last-line">
+    <div class="sponsor-support">
+        <img class="image" src="images/macdonal.jpg" alt="macdonal-logo">
+        <img class="image" src="images/uber.jpg" alt="uber-logo">
+        <img class="image" src="images/KFC.jpg" alt="KFC-logo">
+    </div>
+  </section>
+
+  <footer>
+    <div class="footer">
+      <div class="last-end">
+        <img class="image tof" src="images/main-logo.jpg" alt="restaurant-logo">
+        <h5>@Chef Young  Restaurant Some Rights Reserved</h5>
+      </div>
+    </div>
+  </footer>
```

Comment:
- [ ] Great work on incorporating the requested changes provided by the previous reviewers! I noticed that one of the requirements hasn't been fulfilled just yet. Could you please add the background color to the entire footer? Additionally, it would be wonderful if you could double-check the color of the text.

   ### Your design:
   ![image](https://github.com/Young-Gaius/capstone-project/assets/107952724/8947d5cc-9311-443d-8118-f4b40b1838dd)

   ### Behance template:
   ![image](https://github.com/Young-Gaius/capstone-project/assets/107952724/7c766c98-fa05-4960-87a4-d5d72478d819)